### ISSUE TITLE
Starving 1st layer flow rate

### ISSUE THREAD

**Issue by Rickthebig** [SoftFever/OrcaSlicer#4984]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.0

### Operating System (OS)

Windows

### OS Version

Windows 10

### Printer

BLV Cube

### How to reproduce

1. Slice the retraction calibration model according to the provided project file.
2. No anomaly is seen in the 1st layer flow, but it does not print as it is shown concerning the flow of the 1st layer: It begins with the bottom surface pattern from one corner with overextrusion and ends on the opposite corner with underextrusion.

![2024-04-12 16_41_15-Retraction_test_1st-Layer-anomaly](https://github.com/SoftFever/OrcaSlicer/assets/151161212/692c5544-f439-49e6-a1ef-31308a98fa11)

### Actual results

Flow rate starts too high at the bottom right corner and is gradually decreased towards the opposite left upper corner, where clear gaps are visible due to underextrusion. At midway between both corners, flow rate seems correct. 
All other subsequent layers are OK.

![IMG_20240412_141541](https://github.com/SoftFever/OrcaSlicer/assets/151161212/530bee6a-18a1-4d29-a310-ff0aaf308cd1)

### Expected results

The flow rate should be constant accross the whole layer as it was sliced.

### Project file & Debug log uploads

[Retraction_test_1st-Layer-anomaly.zip](https://github.com/SoftFever/OrcaSlicer/files/14960444/Retraction_test_1st-Layer-anomaly.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Anything else?

Either the slicer is not showing the true extrusion or the firmware of my printer does not like 1st layers of calibration models :-)

**Comment by vgdh** [comment]
do you have the bed leveled correctly?

**Comment by Rickthebig** [comment]
> do you have the bed leveled correctly?

Yes of course. If a 31 cm wide bed would show such a height difference accross 3-4 cm, then it would be very very inclined. But the border line and the brim of the same model are printed perfectly (in this case the brim was turned off). The bed is leveled automatically using a BL Touch prior to every job, and height compensation based on the 100 points heightmap is active.

I have never noticed this to happen with other models, so I think the cause is related to the calibration model itself. By the way, I tried to understand how the retraction is changed with height but I don't see a height modifier in this model...

**Comment by Rickthebig** [comment]
Hereby I am withdrawing this issue submission.
I investigated the issue more thoroughly and found that it is related to a high pressure advance value (using RRF).
I did not notice it before because I was using Cura, allowing to define a PA value for the first layer (which Orca does not) and this 1st layer PA value was set low, so the issue would not appear.
After switching to Orca, I also got a cheap filament, which turns out to be very compressible (PA value 0.98), explaining why this happens.

It is still a bug, but not an Orca's one.

So I am closing this issue.
